src/Symfony/Bridge/Doctrine/Tests/Types/DatePointTypeTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Tests\Types;
+
+use DateTimeImmutable;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Platforms\MariaDBPlatform;
+use Doctrine\DBAL\Platforms\MySQLPlatform;
+use Doctrine\DBAL\Platforms\PostgreSQLPlatform;
+use Doctrine\DBAL\Platforms\SqlitePlatform;
+use Doctrine\DBAL\Types\ConversionException;
+use Doctrine\DBAL\Types\Type;
+use PHPUnit\Framework\TestCase;
+use Symfony\Bridge\Doctrine\Types\DatePointType;
+use Symfony\Component\Clock\DatePoint;
+
+final class DatePointTypeTest extends TestCase
+{
+    private DatePointType $type;
+
+    public static function setUpBeforeClass(): void
+    {
+        $name = DatePointType::NAME;
+        if (Type::hasType($name)) {
+            Type::overrideType($name, DatePointType::class);
+        } else {
+            Type::addType($name, DatePointType::class);
+        }
+    }
+
+    protected function setUp(): void
+    {
+        $this->type = Type::getType(DatePointType::NAME);
+        if (!class_exists(DatePoint::class)) {
```

Comment:
Can be inverted I guess :slightly_smiling_face:

Reply:
I'm not sure to understand how it could be inverted.